README.md

```diff
@@ -163,19 +215,32 @@ produce early errors because of recursive application of the first rule.
         :: MemberExpression
 
 - Let _targetReference_ be the result of evaluating _MemberExpression_.
-- Assert: Type(_targetReference_) is Reference.
+- Assert: Type(_targetReference_) is `Reference`.
 - Let _thisValue_ be GetThisValue(_targetReference_).
 - Let _target_ be GetValue(_targetReference_).
 - If IsCallable(_target_) is `false`, throw a `TypeError` exception.
-- Let _F_ be ? BoundFunctionCreate(_target_, _thisValue_, «»).
-- Let _targetHasLength_ be ? HasOwnProperty(_target_, `"length"`).
-- If _targetHasLength_ is **true**, then
-    - Let _targetLen_ be ? Get(_target_, `"length"`).
-    - If Type(_targetLen_) is not `Number`, then let _L_ be `0`.
-    - Else, let _L_ be ToInteger(_targetLen_).
-- Else let _L_ be `0`.
-- Let _status_ be ? DefinePropertyOrThrow(_F_, `"length"`, PropertyDescriptor {[[Value]]: _L_, [[Writable]]: `false`, [[Enumerable]]: `false`, [[Configurable]]: `true`}).
-- Let _targetName_ be ? Get(_target_, `"name"`).
-- If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
-- Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Let _F_ be BoundFunctionCreate(_target_, _thisValue_, «»).
+- Perform InitBoundProperties(_F_, _target_).
+- Return _F_.
+
+
+----
+
+    BindExpression:
+        LeftHandSideExpression[?Yield] :: new
+
+- Let _targetReference_ be the result of evaluating _LeftHandSideExpression[?Yield]_.
+- Assert: Type(_targetReference_) is `Reference`.
+- Assert: IsSuperReference(_targetReference_) is `false`.
+- Let _target_ be GetValue(_targetReference_).
+- If IsConstructible(_target_) is `false`, throw a `TypeError` exception.
+- Let _bound_ be a new bound constructor wrapper function as defined in Bound Constructor Wrapper Functions.
+- Set the [[BoundTargetConstructor]] internal slot of _bound_ to _target_.
+- Let _desc_ be _target_.\[\[GetOwnProperty\]\]("caller")
```

Comment:
It's just conditionally binding a deprecated property for compatibility. Either this or a strict mode check. If you'd prefer it removed, I have no problem with it, either.